portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -69,6 +71,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     ServletHelper.DEFAULT_DATASTORE_SERVICE.put(commentEntity);
 
     showUploadedComment(response.getWriter(), comment, imageUrl);
+    showImageLabels(request);
```

Comment:
Is it just for debuging?

Reply:
Not really. For now, the tutorial says just print the labels to console in order to make sure the code works. After this step, the labels will be displayed in the webpage.